awx/main/models/schedules.py

```diff
@@ -205,10 +205,15 @@ def rrulestr(cls, rrule, fast_forward=True, **kwargs):
                     'A valid TZID must be provided (e.g., America/New_York)'
                 )
 
-        if fast_forward and ('MINUTELY' in rrule or 'HOURLY' in rrule):
+        if (
+            fast_forward and
+            ('MINUTELY' in rrule or 'HOURLY' in rrule) and
+            'COUNT=' not in rrule
```

Comment:
We really only need this optimization if people set up *unbounded* per minute/hour rules in the past.